torch/csrc/jit/script/compiler.cpp

```diff
@@ -615,7 +661,9 @@ struct to_ir {
   std::vector<Value*> getValues(const Trees& trees) {
     std::vector<Value*> values;
     for (const auto& tree : trees) {
-      values.push_back(emitExpr(tree, 1)[0]);
+      for (Value* v : emitExpr(tree, 1)) {
+        values.push_back(v);
+      }
```

Comment:
Why did this part change? The loop seems unnecessary

Reply:
I think I have to rework how this is done, the invariant that the `emitExpr` call here returns only 1 element is now broken